src/gui/gui.cc

```diff
@@ -1710,6 +1711,18 @@ faster by not displaying the logo.)"));
                          ImGuiInputTextFlags_ReadOnly);
         ImGui::SameLine();
         selectBiosDialog = ImGui::Button("...");
+        
+        auto exp1 = settings.get<Emulator::SettingEXP1>().string();
+        ImGui::InputText(_("EXP1 file"), const_cast<char*>(reinterpret_cast<const char*>(exp1.c_str())), exp1.length(),
+                         ImGuiInputTextFlags_ReadOnly);
+        ImGui::SameLine();
+        selectEXP1Dialog = ImGui::Button("...##EXP1");
+        ImGui::SameLine();
+        if (ImGui::Button(_("Clear"))) {
+            settings.get<Emulator::SettingEXP1>().value = "";
+            changed = true;
+        }
+        
```

Comment:
Might want to add a button to "unload" or "unplug" the exp1 cart, and toggle the "is loaded" boolean accordingly, and rebuild the LUTs.